Formula/csound.rb

```diff
@@ -99,43 +112,68 @@ def install
       resource("ableton-link").stage buildpath/"ableton-link"
       resource("getfem").stage { cp_r "src/gmm", buildpath }
 
-      system "cmake", "-S", ".", "-B", "build",
-                      "-DABLETON_LINK_HOME=#{buildpath}/ableton-link",
-                      "-DBUILD_ABLETON_LINK_OPCODES=ON",
-                      "-DBUILD_CHUA_OPCODES=ON",
-                      "-DBUILD_CUDA_OPCODES=OFF",
-                      "-DBUILD_FAUST_OPCODES=ON",
-                      "-DBUILD_FLUID_OPCODES=ON",
-                      "-DBUILD_HDF5_OPCODES=ON",
-                      "-DBUILD_IMAGE_OPCODES=ON",
-                      "-DBUILD_JACK_OPCODES=ON",
-                      "-DBUILD_LINEAR_ALGEBRA_OPCODES=ON",
-                      "-DBUILD_MP3OUT_OPCODE=ON",
-                      "-DBUILD_OPENCL_OPCODES=OFF",
-                      "-DBUILD_P5GLOVE_OPCODES=ON",
-                      "-DBUILD_PYTHON_OPCODES=ON",
-                      "-DBUILD_STK_OPCODES=ON",
-                      "-DBUILD_WEBSOCKET_OPCODE=ON",
-                      "-DBUILD_WIIMOTE_OPCODES=ON",
-                      "-DCSOUND_FRAMEWORK=#{frameworks}/CsoundLib64.framework",
-                      "-DCSOUND_INCLUDE_DIR=#{include}/csound",
-                      "-DGMM_INCLUDE_DIR=#{buildpath}",
-                      "-DPLUGIN_INSTALL_DIR=#{frameworks}/CsoundLib64.framework/Resources/Opcodes64",
-                      "-DUSE_FLTK=ON",
-                      *std_cmake_args
+      args = [
+        "-DABLETON_LINK_HOME=#{buildpath}/ableton-link",
+        "-DBUILD_ABLETON_LINK_OPCODES=ON",
+        "-DBUILD_CHUA_OPCODES=ON",
+        "-DBUILD_CUDA_OPCODES=OFF",
+        "-DBUILD_FAUST_OPCODES=ON",
+        "-DBUILD_FLUID_OPCODES=ON",
+        "-DBUILD_HDF5_OPCODES=ON",
+        "-DBUILD_IMAGE_OPCODES=ON",
+        "-DBUILD_JACK_OPCODES=ON",
+        "-DBUILD_LINEAR_ALGEBRA_OPCODES=ON",
+        "-DBUILD_MP3OUT_OPCODE=ON",
+        "-DBUILD_OPENCL_OPCODES=OFF",
+        "-DBUILD_PYTHON_OPCODES=ON",
+        "-DBUILD_STK_OPCODES=ON",
+        "-DBUILD_WEBSOCKET_OPCODE=ON",
+        "-DGMM_INCLUDE_DIR=#{buildpath}",
+        "-DUSE_FLTK=ON",
+      ]
+      if OS.mac?
+        args << "-DBUILD_P5GLOVE_OPCODES=ON"
+        args << "-DBUILD_WIIMOTE_OPCODES=ON"
+        args << "-DCSOUND_FRAMEWORK=#{frameworks}/CsoundLib64.framework"
+        args << "-DCSOUND_INCLUDE_DIR=#{include}/csound"
```

Comment:
Is there a reason this is now macOS-only?  This doesn't seem like a platform-specific argument like the others are.

Reply:
* P5 Glove opcodes [depend on a libp5glove library](https://github.com/csound/plugins/blob/a5fd7f194e14833c2bb4f574963613537d6460cb/p5glove/CMakeLists.txt#L5) that’s not generally available on Linux (upstream does something different to build these opcode on macOS). This needn’t be platform specific, but I thought it would be clearer to explicitly indicate that P5 Glove opcodes aren’t being built on Linux.
* Wii remote opcodes depend on [WiiUse](https://github.com/wiiuse/wiiuse), which [does not currently have a Linux bottle](https://github.com/Homebrew/homebrew-core/blob/HEAD/Formula/wiiuse.rb); this case is similar to the P5 Glove opcodes.
* For the last two, by default Csound installs frameworks and includes in non-standard locations on macOS, so when building Csound plug-ins on macOS these locations need to defined.

Reply:
Ah sorry I should have specified that I was only asking about `CSOUND_INCLUDE_DIR`.  It sounds like it doesn't need to be specified on Linux because it already gets installed there.

Eventually we'll probably want to get `libp5glove` and `wiiuse` bottled on Linux, but since we don't currently have bottles there is no reason we have to wait to bottle this, especially given that they are optional.

Reply:
No worries!

Just want to add: `include/"csound"` is a symlink to `frameworks/"CsoundLib64.framework/Headers"`:

https://github.com/Homebrew/homebrew-core/blob/7a13dfdee9cbb5548924e247fa741bcaa944cd7e/Formula/csound.rb#L103

The last 3 CMake arguments could be—

```rb
args << "-DCSOUND_FRAMEWORK=#{frameworks}/CsoundLib64.framework"
args << "-DCSOUND_INCLUDE_DIR=#{frameworks}/CsoundLib64.framework/Headers"
args << "-DPLUGIN_INSTALL_DIR=#{frameworks}/CsoundLib64.framework/Resources/Opcodes64"
```

—which might clarify what’s happening.